src/chips/sam4l/target.json

```diff
@@ -6,5 +6,18 @@
     "arch": "arm",
     "os": "none",
     "morestack": false,
-    "executables": true
+    "executables": true,
+    "no-compiler-rt": true,
+    "relocation-model": "static",
+    "linker": "arm-none-eabi-gcc",
+    "disable-redzeon": true,
```

Comment:
I saw a github email but didn't actually see it in the interface, basically, see http://www.randomhacks.net/2015/11/11/bare-metal-rust-custom-target-kernel-space/

So, this might not actually be needed on your platform, as I think it's x86-specific.... but the dangerous bit is that it will _seem_ like it works, but then not actually work sometimes.